projet/projects/views.py

```diff
@@ -26,11 +26,11 @@
 )
 from rest_framework import status
 from users.serializers import ContributorSerializer
-from users.permissions import IsOwnerOrAdminOfProject, IsContributor, IsOwnerOrReadOnly
```

Comment:
Stopped using rest permissions in some places because of certain inconviniences.
Working on building my own permission custom made for certain views, which will be re-usable.

Reply:
Looking to totally replace the rest permissions with my custom ones except the Is authorized base permission.